### PR TITLE
Search and toasts

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey folks implemented Search feature and Toast messages. Also added dark theme

Live link: https://62854df37d60c22e0a9a4207--quiet-dolphin-319af9.netlify.app/

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 
 import NavBar from "./navbar/Navbar";
-import HomePage from "./homepage/HomePage";
+// import HomePage from "./homepage/HomePage";
 import Sidebar from "./sidebar/Sidebar";
-import LoginPage from "./authpage/LoginPage";
-import SignUpPage from "./authpage/SignUpPage";
-import ExplorePage from "./explorepage/ExplorePage";
-import VideoPage from "./videopage/VideoPage";
+// import LoginPage from "./authpage/LoginPage";
```
Comment (saurabhan) [inline src/components/index.js]: Remove Commented code

**Review by saurabhan — APPROVED** [review]
The code is clean, new file structure is good,
Do remove commented and unused code.

File: src/pages/playlistpage/PlaylistPage.jsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from "react";
+import { VideoCard } from "../../components";
+import { usePlaylist } from "../../context";
+import AddPlaylistModal from "./AddPlaylistModal";
+
+const styleObj = {
+    backgroundColor: "var(--PRIMARY-200)",
+};
+
+export default function PlaylistPage() {
```
Comment (sumitk-singh) [inline src/pages/playlistpage/PlaylistPage.jsx:10]: why are you not using aero function?

File: src/components/videocard/videocard.css [hunk 3]
```diff
@@ -139,6 +139,7 @@
     max-width: 3.2rem;
     height: 3.2rem;
     border-radius: 99px;
```
Comment (sumitk-singh) [inline src/components/videocard/videocard.css]: here, you may use rem unit.

**Review by sumitk-singh — APPROVED** [review]
UI looks great @bhushannaikaroskar ,
Clean and readable code,
Good to merge

File: src/pages/playlistpage/PlaylistPage.jsx (hunk 2 above)
Comment (bhushannaikaroskar) [reply src/pages/playlistpage/PlaylistPage.jsx:10]: I personally like to prefer normal function for Components

**Pull request merged**